internal/server/spanner/query.go

```diff
@@ -58,193 +58,144 @@ var statements = struct {
 	searchNodesByQueryAndTypes string
 }{
 	getPropsBySubjectID: `
-	SELECT
-		DISTINCT subject_id,
-		predicate
-	FROM 
-		Edge
-	WHERE
-		subject_id IN UNNEST(@ids)
-	ORDER BY
-		subject_id,
-		predicate
+		GRAPH DCGraph MATCH -[e:Edge
```

Comment:
Do we have any numbers (or a sense for) on the performance improvements with GQL?

Reply:
It should speed up queries with filters significantly